lib/bloc/redeem_bloc.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:bloc/bloc.dart';
+import 'package:equatable/equatable.dart';
+
+part 'redeem_event.dart';
+part 'redeem_state.dart';
+
+class RedeemBloc extends Bloc<RedeemEvent, RedeemState> {
```

Comment:
If we are going to implement the BLoC for redeem dialog then we should create test for it

Reply:
i have removed all the bloc code. so no need for any test